### PR TITLE
locks expire after 5mins now

### PR DESCRIPTION
**Pull request by jdcanas**

### CODE REVIEW

**Comment by Qi77Qi** [comment]
Shall we just change this to 3 min?

**Comment by calbach** [comment]
I like 3min if there are no objections.

File: server/src/main/resources/reference.conf [hunk 1]
```diff
@@ -3,7 +3,7 @@ path-to-storage-links-json = "/work/.welder/storage_links.json"
 path-to-gcs-metadata-json = "/work/.welder/gcs_metadata.json"
 object-service {
   working-directory = "/work"
-  lock-expiration = 20 minutes
+  lock-expiration = 3 minutes
   owner-email = ${OWNER_EMAIL}
 }
 clean-up-lock-interval = 7 minutes
```
Comment (calbach) [inline server/src/main/resources/reference.conf]: Doesn't this need to be shorter than the lock-expiration?

Comment (Qi77Qi) [reply server/src/main/resources/reference.conf]: No, this is more of nice to have optimization. If it’s shorter than lock expiration, then we’ll be trying to clean up lock more often we need to

**Pull request merged**